kong/runloop/handler.lua

```diff
@@ -204,6 +211,14 @@ local function csv(s)
 end
 
 
+local function try_incr_counter(key)
+  -- key must be DECLARATIVE_ROUTERS_REBUILD_COUNT_KEY or DECLARATIVE_PLUGINS_REBUILD_COUNT_KEY
+  local _, err = kong_shm:incr(key, 1, 0)
+  if err then
+    log(ERR, "failed to incr counter: ", err)
```

Comment:
```suggestion
    log(ERR, "failed to incr router rebuild counter: ", err)
```

Reply:
If `shm:incr` fails, it will cause `not ready` status finally. What shall we do?

Reply:
> If `shm:incr` fails, it will cause `not ready` status finally. What shall we do?

It seems unlikely to fail since we have assigned a number as its initial value. The only possible cause of failure would be a memory exception.